openstack/identity/v3/agency/urls.go

```diff
@@ -1,24 +1,32 @@
 package agency
 
-import "github.com/opentelekomcloud/gophertelekomcloud"
+import (
+	"strings"
+
+	"github.com/opentelekomcloud/gophertelekomcloud"
+)
 
 const (
 	rootPath     = "OS-AGENCY"
 	resourcePath = "agencies"
 )
 
 func rootURL(c *golangsdk.ServiceClient) string {
-	return c.ServiceURL(rootPath, resourcePath)
+	url := c.ServiceURL(rootPath, resourcePath)
+	return strings.Replace(url, "/v3/", "/v3.0/", 1)
```

Comment:
it didn't break anything?

Reply:
Agencies weren't working to begin with, this url change fixed creation/deletion.
I needed that part for testing so I didn't check other functionality

Reply:
I've left agency part as it was before and made some changes to client init, please check